package.json

```diff
@@ -252,7 +252,7 @@
         },
         {
           "command": "openshift.explorer.logout",
-          "when": "view == openshiftProjectExplorer",
+          "when": "view == openshiftProjectExplorer && isLoggedIn",
```

Comment:
should be `context == "isLoggedIn"` otherwise it does not work.

Reply:
It works without setting the context word. Let me see if I can add a gif for it.

Reply:
@dgolovin Updated the description with the gif.